app/views/foreman/job_templates/change_content_source.erb

```diff
@@ -0,0 +1,46 @@
+<%#
+kind: job_template
+name: Change content source
+job_category: Katello
+model: JobTemplate
+provider_type: SSH
+description_format: Configure subscription manager to new content source
+feature: katello_change_content_source
+template_inputs:
+- name: Smart Proxy
+  required: true
+  input_type: user
+  value_type: resource
+  resource_type: SmartProxy
+  description: New content source
+  advanced: false
+%>
+#!/bin/sh
+<%
+    smart_proxy = input_resource('Smart Proxy')
+    rhsm_url = smart_proxy.rhsm_url
+    pulp_content_url = smart_proxy.pulp_content_url
+-%>
+
+SSL_CA_CERT=$(mktemp)
```

Comment:
Why is a temporary file needed?

Reply:
I guess to avoid race condition and to auto-clean up after the use, we use the same concept in the registration if I'm not mistaken